File: src/main/java/club/deneb/client/features/modules/player/Fastbreak.java
package club.deneb.client.features.modules.player;


import club.deneb.client.features.Category;
import club.deneb.client.features.Module;

/**
 * @author 086
 */
@Module.Info(name = "Fastbreak", category = Category.PLAYER, description = "Nullifies block hit delay")
public class Fastbreak extends Module {

    @Override
    public void onTick() {
        mc.playerController.blockHitDelay = 0;
    }
}

File: src/main/java/club/deneb/client/features/modules/player/FastBreak.kt
package club.deneb.client.features.modules.player

import club.deneb.client.features.Category
import club.deneb.client.features.Module

@Module.Info(name = "FastBreak", category = Category.PLAYER, description = "Nullifies block hit delay")
class FastBreak : Module() {
    override fun onTick() {
        mc.playerController.blockHitDelay = 0
    }
}